### PR TITLE
Enable compatibility with Rails 6.1

### PR DESCRIPTION
**Pull request by bquorning**
~Blocked by #27.~
~Blocked by #28.~

When running the tests with Rails 6.0, you would see this deprecation warning:

    DEPRECATION WARNING: Rails 6.1 will return Content-Type header without modification. If you want just the MIME type, please use `#media_type` instead.

For this reason, I have added a `#media_type` helper method at the end of the jsonp_test.rb file to easily work across Rails versions.

### CODE REVIEW

File: test/jsonp_test.rb
Comment (pschambacher) [inline test/jsonp_test.rb:115]: 😬

**Review by pschambacher — APPROVED** [review]

File: gemfiles/rails6.1.gemfile
Comment (mriddle) [inline gemfiles/rails6.1.gemfile:4]: I wasn't sure what the story was here. It looks like `actionpack-action_caching` has Rails 6.1 support but it hasn't been released yet. 

👍

**Review by mriddle — APPROVED** [review]

File: gemfiles/rails6.1.gemfile
Comment (bquorning) [reply gemfiles/rails6.1.gemfile:4]: Exactly. I can try to push for them getting a release out before merging this PR.

Comment (mriddle) [reply gemfiles/rails6.1.gemfile:4]: I don't see a reason for it to block this from being merged, I would ship this and reach out to them.

Comment (bquorning) [reply gemfiles/rails6.1.gemfile:4]: I asked in https://github.com/rails/actionpack-action_caching/pull/77#issuecomment-759345436. Let's see if we don't get a new version in a day or two.

**Pull request merged**
